net/miniupnpd/patches/400-securemode-for-igd-ipv6.patch

```diff
@@ -0,0 +1,11 @@
+--- a/upnpsoap.c
```

Comment:
This patch should be backported here as https://github.com/miniupnp/miniupnp/commit/c79e25a.patch

It will have the Git header to make sure who is the author, subject, description and we know if is backported or not